docs/resources/port_action.md

```diff
@@ -223,7 +223,7 @@ Required:
 
 Optional:
 
-- `payload` (String) The Azure Devops workflow payload (array or object encoded to a string)
+- `payload` (String) The Azure Devops workflow [payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload) (array or object encoded to a string use [jsonencode](https://developer.hashicorp.com/terraform/language/functions/jsonencode)). Learn about how to [define the action payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload).
```

Comment:
https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload link is here twice on purpose?
[payload] => learn about how [define]

Reply:
yes